common/network-common/src/main/java/org/apache/spark/network/server/TransportChannelHandler.java

```diff
@@ -48,7 +47,7 @@
  * on the channel for at least `requestTimeoutMs`. Note that this is duplex traffic; we will not
  * timeout if the client is continuously sending but getting no responses, for simplicity.
  */
-public class TransportChannelHandler extends SimpleChannelInboundHandler<Message> {
+public class TransportChannelHandler extends ChannelInboundHandlerAdapter {
```

Comment:
SimpleChannelInboundHandler also uses Javassist to generate a matcher class. Since `SimpleChannelInboundHandler` provides little value for us, I just changed to extend `ChannelInboundHandlerAdapter` directly.